__tests__/endpoint.test.js

```diff
@@ -87,3 +87,55 @@ describe('GET /api/reviews/:review_id', () => {
     })
   });
 });
+
+describe('GET /api/reviews', () => {
+	test('should return a status code 200', () => {
+		return request(app).get('/api/reviews').expect(200);
+	});
+	test('200: return an object', () => {
+		return request(app)
+			.get('/api/reviews')
+			.then(({ body }) => {
+				expect(typeof body).toBe('object');
+				expect(Array.isArray(body)).toBe(false);
+			});
+	});
+	test('200: should return an array of review objects containing the correct properties', () => {
+		return request(app)
+			.get('/api/reviews')
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+				expect(reviews).toHaveLength(13);
+				reviews.forEach((review) => {
+					expect(review).toMatchObject({
+						owner: expect.any(String),
+						title: expect.any(String),
+						review_id: expect.any(Number),
+						category: expect.any(String),
+						review_img_url: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						designer: expect.any(String),
+						comment_count: expect.any(String),
+					});
+				});
+			});
+	});
+	test('200: should return an array of objects that is sorted by the value of the date property in descending order', () => {
+		return request(app)
+			.get('/api/reviews')
+			.expect(200)
+			.then(({ body }) => {
+				const { reviews } = body;
+        const reviewsCopy = [...reviews];
+        const sortedReviews = reviewsCopy.sort((a,b) => {
+          return b.created_at.localeCompare(a.created_at)
+        });
+        expect(reviews).toEqual(sortedReviews);	        
+			});
+	});
```

Comment:
this is one way to check the order but you could also use `jest-sorted` and i see you have installed it 👀

Reply:
Thanks, Heather! yeah i tried "sorting" jest-sorted but it still wasnt working after following the setup instructions but i will try and have another crack at it!